tests/Keystore/StoredKeyTests.cpp

```diff
@@ -361,8 +361,8 @@ TEST(StoredKey, MissingAddress) {
     auto key = StoredKey::load(TESTS_ROOT + "/Keystore/Data/missing-address.json");
     key.fixAddresses(password);
 
-    EXPECT_EQ(key.account(TWCoinTypeEthereum, nullptr)->address, "0x04De84ec355BAe81b51cD53Fdc8AA30A61872C95");
-    EXPECT_EQ(key.account(coinTypeBc, nullptr)->address, "bc1qe938ncm8fhdqg27xmxd7lq02jz9xh0x48r22lc");
+    EXPECT_EQ(key.account(TWCoinTypeEthereum, nullptr)->address, "0xA3Dcd899C0f3832DFDFed9479a9d828c6A4EB2A7");
```

Comment:
can you add your test phrases to comment? next time we won't have to change existing test data

Reply:
The test data contained the test mnemonic with `back` as the last word, which is not a valid BIP39 word (worked before w/o check for valid mnemo).
I added now explicit check for the mnemonic in the loaded wallet.